src/components/RouteInputForm/RouteInputForm.stories.jsx

```diff
@@ -8,51 +8,59 @@ export default {
   tags: ["autodocs"],
 };
 
-export const Default = {
+export const Row = {
   args: {
-    startingPointValue: null,
     onStartingPointChange: fn(),
-    destinationValue: null,
     onDestinationChange: fn(),
-    arriveByDateTimeValue: "2023-09-10T10:00",
     onArriveByDateTimeChange: fn(),
+    arriveByDateTimeValue: "2023-09-10T10:00",
   },
   render: function (args) {
-    const [startingPointValue, setStartingPointValue] = useState(
-      args.startingPointValue,
+    const [arriveByDateTime, setArriveByDateTime] = useState(
+      args.arriveByDateTimeValue,
     );
 
-    const [destinationValue, setDestinationValue] = useState(
-      args.destinationValue,
+    function onArriveByDateTimeChange(datetime) {
+      args.onArriveByDateTimeChange(datetime); // fn()
+      setArriveByDateTime(datetime);
+    }
+
+    return (
+      <RouteInputForm
+        onStartingPointChange={args.onStartingPointChange}
+        onDestinationChange={args.onDestinationChange}
+        onArriveByDateTimeChange={onArriveByDateTimeChange}
+        arriveByDateTimeValue={arriveByDateTime}
+        orientation="row"
+      />
     );
+  },
+};
 
+export const Column = {
+  args: {
+    onStartingPointChange: fn(),
+    onDestinationChange: fn(),
+    onArriveByDateTimeChange: fn(),
+    arriveByDateTimeValue: "2023-09-10T10:00",
+  },
+  render: function (args) {
     const [arriveByDateTime, setArriveByDateTime] = useState(
       args.arriveByDateTimeValue,
     );
 
-    function onStartingPointChange(location) {
-      args.onStartingPointChange(location); // fn() for debugging
-      setStartingPointValue(location);
-    }
-
-    function onDestinationChange(location) {
-      args.onDestinationChange(location); // fn() for debugging
-      setDestinationValue(location);
-    }
-
     function onArriveByDateTimeChange(datetime) {
-      args.onArriveByDateTimeChange(datetime); // fn() for debugging
+      args.onArriveByDateTimeChange(datetime); // fn()
       setArriveByDateTime(datetime);
     }
 
     return (
       <RouteInputForm
-        startingPointValue={startingPointValue}
-        onStartingPointChange={onStartingPointChange}
-        destinationValue={destinationValue}
-        onDestinationChange={onDestinationChange}
-        arriveByDateTimeValue={arriveByDateTime}
+        onStartingPointChange={args.onStartingPointChange}
+        onDestinationChange={args.onDestinationChange}
         onArriveByDateTimeChange={onArriveByDateTimeChange}
+        arriveByDateTimeValue={arriveByDateTime}
+        orientation="column"
```

Comment:
i suspect this to be a merge issue of some sorts - please add the old values back and use the old order to mirror the order in which the properties are defined in the RouteInputForm component